verbsense/pom.xml

```diff
@@ -0,0 +1,173 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>edu.illinois.cs.cogcomp</groupId>
+	<artifactId>illinois-verbsense</artifactId>
+	<packaging>jar</packaging>
+	<version>1.2</version>
+
```

Comment:
look at the `pom.xml` file of other modules and see how they look. Specifically you have to include information  about the parent module.